### PR TITLE
Add logo to top of main README, rearranging sections

### PR DESCRIPTION
**Pull request by hl662**
Contributing to improve the repo's presentation to new users

- Add a .svg icon that has clearer contrast for use.
- moved licensing and contribution headers to the bottom of the README.
- Added badges to highlight min Node supported versions, and latest versions of core packages

For anyone wanting to replicate the badges, I used [shields.io](https://shields.io/badges/dynamic-json-badge), which provides free APIs to parse a JSON file and use one of it's properties as a badge value.

Screenshot of current logo on dark mode: 
image
Screenshot of current logo on light mode: 
image

### CODE REVIEW

File: docs/assets/imodeljs.svg
Comment (nick4598) [inline docs/assets/imodeljs.svg:1]: Should the name of this be changed?

**Comment by nick4598** [comment]
Can you post a screenshot of what the readme looks like now?

File: docs/assets/imodeljs.svg
Comment (hl662) [reply docs/assets/imodeljs.svg:1]: ah, lemme do that

**Comment by pmconne** [comment]
Consider adding a link to the itwinjs.org doc site for people who end up here with no idea what iTwin.js is for or how to use it.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
-﻿# iTwin.js
-
-Copyright © Bentley Systems, Incorporated. All rights reserved. See [LICENSE.md](./LICENSE.md) for license terms and full copyright notice.
-
-[iTwin.js](http://www.itwinjs.org) is an open source platform for creating, querying, modifying, and displaying Infrastructure Digital Twins.
-
-If you have questions, or wish to contribute to iTwin.js, see our [Contributing guide](./CONTRIBUTING.md).
+﻿<p align="center">
```
Comment (ben-polinsky) [inline README.md:1]:
```suggestion
<p style="display: flex; justify-content: center; align-items: center;">
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,22 @@
...
+ <a href="https://www.itwinjs.org/" target="_blank" rel="noopener noreferrer">
+    <img width="180" src="./docs/assets/imodeljs.svg" alt="iTwin.js Logo">
+  </a>
+</p>
+
+<h1 align="center">
```
Comment (ben-polinsky) [inline README.md:7]:
```suggestion
<h1 style="text-align: center;">
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,22 @@
...
+iTwin.js
+</h1>
+<p align="center">
```
Comment (ben-polinsky) [inline README.md:10]:
```suggestion
<p style="text-align: center;">
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,22 @@
...
+    <a href="https://github.com/iTwin/itwinjs-core/releases/latest"><img src="https://img.shields.io/github/v/release/iTwin/itwinjs-core?label=latest" alt="Latest version"></a>
+    <a href="https://nodejs.org/en/about/previous-releases">
+    <img src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fgithub.com%2FiTwin%2Fitwinjs-core%2Fraw%2Fmaster%2Frush.json&query=%24.nodeSupportedVersionRange&label=node&color=33a921"
+    alt="Supported Node Versions">
+    </a>
+</a>
+<p align="center">
```
Comment (ben-polinsky) [inline README.md:17]:
```suggestion
<p style="text-align: center;">
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,22 @@
...
+An open source platform for creating, querying, modifying, and displaying Infrastructure Digital Twins.
```
[the remark this answers is not in the document]
Comment (MichaelSwigerAtBentley) [reply README.md:18]: Makes sense. Can hold off for now.

File: docs/assets/imodeljs.svg
Comment (hl662) [reply docs/assets/imodeljs.svg:1]: done

**Comment by ben-polinsky** [comment]
Welp, apparently gh doesn't allow css and the deprecated `align` property is the way to center... apologies. https://github.com/orgs/community/discussions/22728

**Comment by hl662** [comment]
> Can you post a screenshot of what the readme looks like now?

Added screenshots to the PR description

**Review by MichaelSwigerAtBentley — APPROVED** [review]

File: .vscode/settings.json [hunk 6]
```diff
@@ -83,7 +83,9 @@
   "markdownlint.config": {
     "MD024": {
       "allow_different_nesting": true
-    }
+    },
+    "MD033": false,
```
Comment (aruniverse) [inline .vscode/settings.json:87]: fyi for those interested

https://github.com/DavidAnson/markdownlint/blob/main/doc/md033.md
https://github.com/DavidAnson/markdownlint/blob/main/doc/md041.md

File: README.md [hunk 7]
```diff
@@ -1,10 +1,22 @@
...
```
Comment (aruniverse) [inline README.md:18]:
```suggestion
An open source library for creating, querying, modifying, and displaying Infrastructure Digital Twins.
```

File: README.md [hunk 8]
```diff
@@ -1,10 +1,22 @@
...
+  </a>
+</p>
+
+<h1 align="center">
+iTwin.js
+</h1>
+<p align="center">
+    <a href="https://github.com/iTwin/itwinjs-core/releases/latest"><img src="https://img.shields.io/github/v/release/iTwin/itwinjs-core?label=latest" alt="Latest version"></a>
+    <a href="https://nodejs.org/en/about/previous-releases">
+    <img src="https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fgithub.com%2FiTwin%2Fitwinjs-core%2Fraw%2Fmaster%2Frush.json&query=%24.nodeSupportedVersionRange&label=node&color=33a921"
```
Comment (aruniverse) [inline README.md:13]: how did you create this badge?
mention this in the description

Comment (hl662) [reply README.md:13]: okay, updated description

**Review by aruniverse — APPROVED** [review]
good first pass, we will need to iteratively improve this over time

**Pull request merged**